website/docs/utils/neighbor-location/examples/grouped-input-and-button.tsx

```diff
@@ -8,23 +8,28 @@ const Demo = () => {
     <>
       <Flex mb={4}>
         <Input neighborLocation='right' w={200}>
-          <Input.Value placeholder='Placeholder' />
+          <Input.Value placeholder='Placeholder' aria-label={'Input with default button'} />
         </Input>
         <Button neighborLocation='left'>Button</Button>
       </Flex>
       <Flex mb={4}>
         <Input neighborLocation='right' w={200}>
           <Input.Value placeholder='Placeholder' />
         </Input>
-        <Button neighborLocation='left' use='primary'>
+        <Button neighborLocation='left' use='primary' aria-label={'Input with primary button'}>
```

Comment:
It seems like the Aria label ran away from the right place 😁 I will correct in my 
 small commit.